index.html

```diff
@@ -2,12 +2,275 @@
+                <div class="white--text">
+                  Appreciate various creations applying CC license of artists,
+                  organized from Art Center Nabi.
+                </div>
+              </li>
+              <li
+                class="d-grid pa-4 d-md-flex flex-md-column gap-md-1 activity"
+              >
+                <div><span class="icon icon-forum-outline icon-4x"></span></div>
+                <div class="secondary--text font-size-h5">Forum</div>
+                <div class="white--text">
+                  Have the time to share your thoughts and opinions with experts
+                  for each topic.
+                </div>
+              </li>
+              <li
+                class="d-grid pa-4 d-md-flex flex-md-column gap-md-1 activity"
+              >
+                <div>
+                  <span class="icon icon-monitor-dashboard icon-4x"></span>
+                </div>
+                <div class="secondary--text font-size-h5">Whorkshop</div>
+                <div class="white--text">
+                  Try creating your own work using open source license rather
+                  than just watching.
+                </div>
+              </li>
+              <li
+                class="d-grid pa-4 d-md-flex flex-md-column gap-md-1 activity"
+              >
+                <div><span class="icon icon-dots-circle icon-4x"></span></div>
+                <div class="secondary--text font-size-h5">CC Ignite</div>
+                <div class="white--text">
+                  Get opportunities to network with CC affiliates around the
+                  world, also after the summit.
+                </div>
+              </li>
+            </ul>
+          </div>
+
+          <div class="d-flex justify-center align-center buttons">
+            <button type="button" class="btn btn-primary">
+              Join VueConf Summit 2021
+            </button>
+          </div>
+        </div>
+      </section>
+
+      <section class="section featured-speakers" id="speakers">
+        <div class="container">
+          <h2 class="font-size-h2 text-center title">Featured Speakers</h2>
+
+          <div class="speakers"></div>
+
+          <div
+            class="
+              d-flex
+              justify-center
+              align-center
+              w-100
+              pa-4
+              d-md-none
+              buttons
+            "
+          >
+            <button
+              type="button"
+              class="
+                btn btn-light
+                w-100
+                d-flex
+                justify-center
+                align-center
+                text-uppercase
+              "
+            >
+              More
+              <span
+                class="icon icon-chevron-down-secondary icon-2x"
+                aria-hidden
+              ></span>
+            </button>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="footer"></footer>
```

Comment:
- [ ] This project contains a footer, even though the content is the same for both desktop and mobile versions (a logo aligned to the left and a text aligned to the right), they look a bit different on both versions. On the mobile it has a white background, while in desktop it has a dark background.
- [ ] Desktop version:

![image](https://user-images.githubusercontent.com/66967274/142067597-d4a06082-a67d-496e-8610-ba3c919b43fc.png)

- [ ] Mobile version:

![image](https://user-images.githubusercontent.com/66967274/142067638-200984b9-00dd-4765-88b2-47f12b586a32.png)

Reply:
I checked. The dark color is for about page.